src/Sulu/Bundle/MediaBundle/Media/ImageConverter/TransformationPool.php

```diff
@@ -23,8 +24,14 @@ class TransformationPool implements TransformationPoolInterface
      */
     private $transformations = [];
 
+    public function __construct(private ContainerInterface $container)
+    {
+    }
+
```

Comment:
Needs be nullable for bc reasons:

```suggestion
    public function __construct(private ?ContainerInterface $container = null)
    {
    }

```